.github/workflows/slack-pr-notifications.yaml

```
@@ -0,0 +1,754 @@
+            core.setOutput('review_color', reviewColor);
+            core.setOutput('comment_count', reviewComments.data ? reviewComments.data.length.toString() : '0');
+            
+            return {
+              reviewer: review.user.login,
+              reviewState: review.state,
+              commentCount: reviewComments.data ? reviewComments.data.length : 0
+            };
+
+      - name: Send Slack notification for review
+        if: steps.review-details.outputs.skip_notification != 'true'
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)